src/main/java/frc/robot/subsystems/elevator/Elevator.java

```diff
@@ -102,28 +111,28 @@ public void periodic() {
     } else {
       profileTimer.start();
     }
-    if (profileTimer.isRunning()) {
-
-      Rotation2d targetRotation =
-          Rotation2d.fromRotations(
-              profile.calculate(
-                      profileTimer.getTimestamp(),
-                      new State(
-                          this.elevator.getEncoder().getPosition(),
-                          this.elevator.getEncoder().getVelocity()),
-                      new State(
-                          ElevatorPositionNormalized.toRange(
-                              RobotConstants.ElevatorConstants.encoderLowerLimit,
-                              RobotConstants.ElevatorConstants.encoderUpperLimit),
-                          0))
-                  .position);
-
-      elevator.setTargetPosition(
-          BasePosition.fromRange(
-              RobotConstants.ElevatorConstants.encoderLowerLimit,
-              RobotConstants.ElevatorConstants.encoderUpperLimit,
-              targetRotation.getRotations()));
-    }
+    // if (profileTimer.isRunning()) {
+
+    Rotation2d targetRotation =
+        Rotation2d.fromRotations(
+            profile.calculate(
+                    profileTimer.getTimestamp(),
+                    new State(
+                        this.elevator.getEncoder().getPosition(),
+                        this.elevator.getEncoder().getVelocity()),
+                    new State(
+                        ElevatorPositionNormalized.toRange(
+                            RobotConstants.ElevatorConstants.encoderLowerLimit,
+                            RobotConstants.ElevatorConstants.encoderUpperLimit),
+                        0))
+                .position);
+
+    elevator.setTargetPosition(
+        BasePosition.fromRange(
+            RobotConstants.ElevatorConstants.encoderLowerLimit,
+            RobotConstants.ElevatorConstants.encoderUpperLimit,
+            targetRotation.getRotations()));
+    // }
```

Comment:
This seems completely odd. We are running a trapezoidal profile to determine the best way to get to a specified position, but only using it to set a target position? This means we aren't actually driving the motor based on the constraints, how the motor is driven is done in the state handler.

We need to redo this code fully I think.